src/style/shadows.scss

```diff
@@ -32,36 +32,36 @@
-        --button-shadow-pressed: 0 #{pxToRem(1)} #{pxToRem(3)} 0
-                rgba(0, 0, 0, 0.232),
-            0 #{pxToRem(0.3)} #{pxToRem(0.9)} 0 rgba(0, 0, 0, 0.208);
-        --button-shadow-inset: 0 #{pxToRem(0.5)} #{pxToRem(3.5)} 0
-                rgba(0, 0, 0, 0.232) inset,
-            0 #{pxToRem(0.3)} #{pxToRem(0.9)} 0 rgba(0, 0, 0, 0.208) inset,
-            0 0 #{pxToRem(3)} rgba(0, 0, 0, 0.9);
-        --button-shadow-inset-pressed: 0 #{pxToRem(0.5)} #{pxToRem(5.5)} 0
-                rgba(0, 0, 0, 0.232) inset,
-            0 #{pxToRem(0.3)} #{pxToRem(1.9)} 0 rgba(0, 0, 0, 0.28) inset,
-            0 0 #{pxToRem(3)} rgba(255, 255, 255, 0.9);
-
-        --shadow-depth-8: 0 #{pxToRem(3.2)} #{pxToRem(7.2)} 0
-                rgba(0, 0, 0, 0.232),
-            0 #{pxToRem(0.6)} #{pxToRem(1.8)} 0 rgba(0, 0, 0, 0.208);
-
-        --shadow-depth-16: 0 #{pxToRem(6.4)} #{pxToRem(14.4)} 0
-                rgba(0, 0, 0, 0.232),
-            0 #{pxToRem(1.2)} #{pxToRem(3.6)} 0 rgba(0, 0, 0, 0.208);
-
-        --shadow-depth-64: 0 #{pxToRem(25.6)} #{pxToRem(57.6)} 0
-                rgba(0, 0, 0, 0.32),
-            0 #{pxToRem(4.8)} #{pxToRem(14.4)} 0 rgba(0, 0, 0, 0.18);
-    }
+    // @include in(dark-mode) {
+    //     // dark mode versions of the above
+    //     --button-shadow-normal: 0 #{pxToRem(1.5)} #{pxToRem(3.6)} 0
+    //             rgba(0, 0, 0, 0.132),
+    //         0 #{pxToRem(0.3)} #{pxToRem(0.9)} 0 rgba(0, 0, 0, 0.108);
+    //     --button-shadow-hovered: 0 #{pxToRem(2)} #{pxToRem(6)}
+    //             rgba(0, 0, 0, 0.25),
+    //         0 #{pxToRem(6)} #{pxToRem(10)} rgba(0, 0, 0, 0.15);
+    //     --button-shadow-pressed: 0 #{pxToRem(1)} #{pxToRem(3)} 0
+    //             rgba(0, 0, 0, 0.232),
+    //         0 #{pxToRem(0.3)} #{pxToRem(0.9)} 0 rgba(0, 0, 0, 0.208);
+    //     --button-shadow-inset: 0 #{pxToRem(0.5)} #{pxToRem(3.5)} 0
+    //             rgba(0, 0, 0, 0.232) inset,
+    //         0 #{pxToRem(0.3)} #{pxToRem(0.9)} 0 rgba(0, 0, 0, 0.208) inset,
+    //         0 0 #{pxToRem(4)} rgba(255, 255, 255, 0.9);
+    //     --button-shadow-inset-pressed: 0 #{pxToRem(0.5)} #{pxToRem(5.5)} 0
+    //             rgba(0, 0, 0, 0.232) inset,
+    //         0 #{pxToRem(0.3)} #{pxToRem(1.9)} 0 rgba(0, 0, 0, 0.28) inset,
+    //         0 0 #{pxToRem(3)} rgba(255, 255, 255, 0.9);
+    //
+    //     --shadow-depth-8: 0 #{pxToRem(3.2)} #{pxToRem(7.2)} 0
+    //             rgba(0, 0, 0, 0.232),
+    //         0 #{pxToRem(0.6)} #{pxToRem(1.8)} 0 rgba(0, 0, 0, 0.208);
+    //
+    //     --shadow-depth-16: 0 #{pxToRem(6.4)} #{pxToRem(14.4)} 0
+    //             rgba(0, 0, 0, 0.232),
+    //         0 #{pxToRem(1.2)} #{pxToRem(3.6)} 0 rgba(0, 0, 0, 0.208);
+    //
+    //     --shadow-depth-64: 0 #{pxToRem(25.6)} #{pxToRem(57.6)} 0
+    //             rgba(0, 0, 0, 0.32),
+    //         0 #{pxToRem(4.8)} #{pxToRem(14.4)} 0 rgba(0, 0, 0, 0.18);
+    // }
```

Comment:
@Kiarokh

Reply:
@jgroth @BregenzerK I need them to be there. We will used them soon when I start working on colors and dark mode and so forth. The best place to keep them is inside the code. 

I just didn't want them to go out, because they get applied in the dark mode now and affect the design. I have not had the time to fine-tune them, and our UI is only working for light-mode now.

Can we just let them be there?

Reply:
Can you not save them somewhere else? :smile: 
It's not very good practice to keep commented code lying around :slightly_smiling_face:

Reply:
Sure :) I have saved them now locally. @BregenzerK will you remove them while you are working on the branch? I do you want me to push now?

Reply:
yes sure, I can remove it!